gcp_variant_transforms/libs/variant_partition.py

```diff
@@ -90,3 +289,29 @@ def _get_auto_partition(self, reference_name):
       # Save partition in _reference_name_to_partition dict for future lookups
       self._reference_name_to_partition_map[reference_name] = partition
```

Comment:
Can we use the same structure for both auto and config file? i.e. map of chr to Region class.

Reply:
As we discussed I replaced _reference_name_to_partition_map and now in both modes we are using the same dict.